BetterMD/__init__.py

```
@@ -1,10 +1,48 @@
 import logging
-from .elements import A, H1, H2, H3, H4, H5, H6, Head, OL, UL, LI, Text, Div, P, Span, Img, B, I, Br, Blockquote, Hr, Table, Tr, Td, Th, THead, TBody, Input, Code
+from .elements import *
 from .html import CustomHTML
 from .markdown import CustomMarkdown
 from .rst import CustomRst
+from .parse import HTMLParser, MDParser, Collection
 
+class HTML:
+    @staticmethod
+    def from_string(html:'str'):
+        return Symbol.from_html(html)
```

Ruff (0.8.2)

`Symbol` may be undefined, or defined from star imports

(F405)